addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -500,11 +492,17 @@ function ThreadFactory({ Entity }) {
          * @private
          * @returns {boolean}
          */
-        _computeIsModeratedByUser() {
+        _computeIsModeratedByCurrentPartner() {
             if (this.model !== 'mail.channel') {
                 return false;
             }
-            return Thread.moderatedChannelIds.includes(this.id);
+            if (!this.messaging) {
```

Comment:
This condition should ideally never be necessary.

Reply:
It is just because the order of computes?
Maybe then we should use `this.env.messaging` so avoid the condition here?
This is very related to the similar comment about the messaging in chat window manager.

Reply:
yes, but this is required for the moment because this is done from a compute, and order of compute is arbitrary